keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/UnhappyVetoRepository.kt

```diff
@@ -40,7 +40,7 @@ abstract class UnhappyVetoRepository(
   abstract fun markUnhappyForWaitingTime(
     resourceId: String,
     application: String,
-    wait: Duration = Duration.parse(waitingTime)
+    wait: Duration? = Duration.parse(waitingTime)
```

Comment:
Probably worth a comment in the function header that passing `null` for the wait means that it is unhappy forever.